packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples/FixedWidthColumns.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import CompactInteractiveList, { alignTypes } from 'terra-compact-interactive-list';
+import rows from './rowsData';
```

Comment:
In order to make the examples more useful, all relevant implementation information should be included in the example's jsx file. For components like this, the data fed into it is important so it should be included, even if that means repeating it across examples.

In this instance, the focus of the example is the columns so the row data may not be terribly important, but there should be at least one base example that includes the row data inline.

Reply:
Fixed in https://github.com/cerner/terra-framework/pull/1869/commits/a60c1c52389d252309f3aeaed4b1de21ace9f470